src/Abstracts/MergeTag.php

```diff
@@ -212,6 +212,24 @@ public function set_resolver( $resolver ) {
 
 	}
 
+	/**
+	 * Sets resolver function
+	 *
+	 * @since 8.0.11
```

Comment:
```suggestion
	 * @since [Next]
```

Check other docblocks